dbt_copilot_helper/templates/pipelines/codebase/overrides/buildspec.deploy.yml

```diff
@@ -0,0 +1,40 @@
+version: 0.2
+
+env:
+  git-credential-helper: yes
+  variables:
+    COLOR: false
```

Comment:
Because the codebuild job doesn't parse ansi escape codes for colors so we would end up with `\u001b[31mSome warning text\u001b[0m` or similar in the logs.